zerver/tests/test_custom_profile_data.py

```diff
@@ -888,6 +964,94 @@ def test_default_external_account_type_field(self) -> None:
         )
         self.assert_json_error(result, "Default custom field cannot be updated.")
 
+    def test_update_with_editable_by_user(self) -> None:
+        iago = self.example_user("iago")
+        aaron = self.example_user("aaron")
+
+        self.login("iago")
+
+        # Create field with editable_by_user = false
+        data: Dict[str, Any] = {}
+        data["name"] = "Name "
```

Comment:
Why the trailing whitespace here? It looks like a bug, and likely isn't what you want to test...

Reply:
Sorry, I did not think of issues due to passing unstripped whitespace to backend function.

However, as pointed above, passing `name` is no longer neccesary to modify profile fields.